example/ios/Podfile

```diff
@@ -1,7 +1,7 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, min_ios_version_supported
+platform :ios, '14.0' # get help here 🙆‍♂️
```

Comment:
Since this is for the Example App, hardcoding 14.0 is not a big deal.
However, could you delete the comment?